app/assets/javascripts/client/common_create/CreateService.js

```diff
@@ -6,16 +6,35 @@
   CreateService.$inject = [
     '$window',
     '$location',
-    'Assessment'
+    '$stateParams',
+    'Assessment',
+    'Inventory',
+    'Participant',
+    'InventoryParticipant'
   ];
 
-  function CreateService($window, $location, Assessment) {
+  function CreateService($window, $location, $stateParams, Assessment, Inventory, Participant, InventoryParticipant) {
```

Comment:
I hadn't settled on the "tool" naming convention until a fair bit after this service had been created.  I'm on board for renaming it, but it is something that we could tackle in subsequent cleanup work?